frontend/libs/studio-components/src/components/StudioInputTable/Cell/BaseInputCell.ts

```diff
@@ -41,13 +41,35 @@ export abstract class BaseInputCell<
   };
 
   private handleKeyDown(event: KeyboardEvent<Element>): void {
+    switch (event.key) {
+      case 'ArrowUp':
+      case 'ArrowDown':
+      case 'ArrowLeft':
+      case 'ArrowRight':
+        this.handleArrowKeyDown(event);
+        break;
```

Comment:
Probably correct implementation, but the naming makes it confusing that all the arrow cases are leading to `arrowKeyDown`?

Reply:
Good point. Do you have any suggestion for a better name? `KeyDown` is there to refer to the `keyDown` event (and to distinguish it from the `keyUp` event).

Reply:
I'm not confident in what actually happens here so I am not sue 🤔

Reply:
Probably not much to do here to improve the readability sine the event is called keyDown 🤷‍♀️